docs/releases/1.11.29.1.txt

```diff
@@ -0,0 +1,30 @@
+============================
+Django 1.11.29 release notes
+============================
+
+*November 2, 2021*
+
+Django 1.11.29 security fixes
+
+CVE-2021-33203: Potential directory traversal via ``admindocs``
+===============================================================
+
+Staff members could use the :mod:`~django.contrib.admindocs`
+``TemplateDetailView`` view to check the existence of arbitrary files.
+Additionally, if (and only if) the default admindocs templates have been
+customized by the developers to also expose the file contents, then not only
+the existence but also the file contents would have been exposed.
+
+As a mitigation, path sanitation is now applied and only files within the
+template root directories can be loaded.
+
+
+*March 4, 2020*
```

Comment:
This already exists in the 1.11.29.txt release note and doesn't need repeating.

Reply:
Yeah, that was an oops.